### ISSUE TITLE
Howto: Batch deleting found duplicate files

### ISSUE THREAD

**Issue by schorfi** [PaulLereverend/NextcloudDuplicateFinder#94]
Hello,
I am facing a huge amount of duplicates due to some redundant upload of files from users after the files got sorted and archived.
The UI of the duplicate finder app is providing only limited progress (just 40 files, one click one action) - esp if facing potential 20k of duplicated files.

So, I was asking myself whether it is possible to delete found duplicates based on a rule (e.g. path prefix to delete preferred on upload directories). This could be in the UI or in the command line. For the latter how could this be done or how can I get the output for all found files  which I could process in a batch/shell/python script for automated deletion.

Is there an existing way to do that?

**Comment by MarkoPaasila** [comment]
Sync to a big enough ssd, then run rmlint.

**Comment by ashish-khokhar** [comment]
@MarkoPaasila Yes that's one of the options, However, we need this feature in this plugin, so I think your suggestion is out of context.
